### DISCUSSION TITLE
PBE+VDW Data training

### DISCUSSION THREAD

**Discussion by haidi-ustc** [CederGroupHub/chgnet#206] [Announcements]
Hello CHGNet team,

I have a set of PBE+VDW data that I would like to use for new training with CHGNet. I would appreciate it if you could provide a training script tailored for this type of data.

Additionally, I have questions regarding the handling of atom_ref in the training process:

How should AtomRef be managed or adjusted when training on PBE+VDW data? Are there specific considerations or best practices I should follow to ensure that the model correctly learns from the provided dataset?
Are there any recommended preprocessing steps or adjustments needed specifically for PBE+VDW data before training?
Thank you in advance for your assistance! Your guidance will be invaluable as I proceed with this task.

Best regards

**Comment by bowen-bd** [comment]
Hi There!
If your dataset size is larger than 1000 structure, then it's OK to follow the standard fine-tuning procedure.
Typically VDW correction does not bring a large elemental energy difference to AtomRef, 
in such case you can skip refitting the AtomRef.
